rst2pdf/findfonts.py

```diff
@@ -222,7 +222,7 @@ def get_variants(family):
         # ctypes with EnumFontFamiliesEx
 
         def get_nt_fname(ftname):
-            import _winreg as _w
+            import winreg as _w
```

Comment:
I think this will break Python 2 on Windows.

How about something like:

```
if six.PY3:
    import winreg as _w
else:
    import _winreg as _w
```
?